cloudrun-malware-scanner/server.js

```diff
@@ -167,26 +159,18 @@ async function handleGcsObject(req, res) {
     if (clamd.isCleanReply(result)) {
       logger.info(`Scan status for gs://${file.bucket}/${file.name}: CLEAN (${
         fileSize} bytes in ${scanDuration} ms)`);
-      metrics.writeScanClean(config.unscanned, config.clean, fileSize,
+      metrics.writeScanClean("gs://${file.bucket}/${file.name}", fileSize,
           scanDuration, clamdVersion);
 
-      // Move document to the bucket that holds clean documents. This can
-      // fail due to permissions or if the file has been deleted.
-      await moveProcessedFile(file.name, true, config);
-
       // Respond to API client.
       res.json({status: 'clean', clam_version: clamdVersion});
     } else {
       logger.warn(`Scan status for gs://${file.bucket}/${
         file.name}: INFECTED ${result} (${
         fileSize} bytes in ${scanDuration} ms)`);
-      metrics.writeScanInfected(config.unscanned, config.quarantined, fileSize,
+      metrics.writeScanInfected("gs://${file.bucket}/${file.name}", fileSize,
```

Comment:
u cant have file.name in the metrics 
the cardinality is too high

Reply:
That's true. Do we want to separate the metric in bucket? or just put them together?

Reply:
by bucket is fine for now

Reply:
Done.